spec.html

```diff
@@ -32359,12 +32345,14 @@ <h1>
           1. If _year_ is not finite or _month_ is not finite or _date_ is not finite, return *NaN*.
           1. Let _y_ be 𝔽(! ToIntegerOrInfinity(_year_)).
           1. Let _m_ be 𝔽(! ToIntegerOrInfinity(_month_)).
```

Comment:
You could consider changing _m_ to a mathematical value, since we only take ℝ(_m_) below anyway. But maybe the current way is more consistent, I'm not sure.